backend/src/main/java/utils/WordPhoneticGetter.java

```
@@ -0,0 +1,79 @@
+    	current += 1;
+    	if (current == 499) {
+			current = 0;
+			try {
+				Thread.sleep(60000);
+			} catch(InterruptedException ex) {
+				Thread.currentThread().interrupt();
+			}
+		}
+
+        try {
+            String json = sendGET(word);
+            List<LinkedHashMap> phonetic = JsonPath.read(json, "$.results[0].lexicalEntries[0].pronunciations");
+
+            for (LinkedHashMap lhm : phonetic) {
+            	if (lhm.get("phoneticNotation") != null && lhm.get("phoneticNotation").equals("IPA")) {
+            		return (String) lhm.get("phoneticSpelling");
+				}
+			}
+
+            return "";
+        } catch (IOException | PathNotFoundException ex) {
+            System.err.println("IOException thrown!");
+            ex.printStackTrace();
+            return "";
+        }
+    }
+
+    private static String sendGET(String word) throws IOException {
+        URL obj = new URL(GET_URL + word.toLowerCase() + "?" + PRONUNCIATION_FIELD);
+        HttpURLConnection con = (HttpURLConnection) obj.openConnection();
+
+        con.setRequestMethod("GET");
+        con.setRequestProperty("User-Agent", USER_AGENT);
+		con.setRequestProperty("Accept","application/json");
+        con.setRequestProperty("app_id", Config.APP_ID.getValue());
+        con.setRequestProperty("app_key", Config.APP_KEY.getValue());
+
+        int responseCode = con.getResponseCode();
+        System.out.println("GET Response Code :: " + responseCode);
+
+        if (responseCode == HttpURLConnection.HTTP_OK) {
+            BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream()));
+            String inputLine;
+            StringBuffer response = new StringBuffer();
+
+            while ((inputLine = in.readLine()) != null) {
+                response.append(inputLine);
+            }
+            in.close();
+
+            return response.toString();
+        } else {
+            return "{ \"results\": [] }";
```

Same here about failing silently. Should we return a 500 to the client so the client knows the backend is being rate limited (or there's some other error)?